samples/dnn/virtual_try_on.py

```diff
@@ -0,0 +1,464 @@
+import argparse
+import math
+import numpy as np
+import cv2 as cv
+
+from numpy import linalg
+from human_parsing import parse_human
+
+backends = (cv.dnn.DNN_BACKEND_DEFAULT, cv.dnn.DNN_BACKEND_INFERENCE_ENGINE, cv.dnn.DNN_BACKEND_OPENCV)
+targets = (cv.dnn.DNN_TARGET_CPU, cv.dnn.DNN_TARGET_OPENCL, cv.dnn.DNN_TARGET_OPENCL_FP16, cv.dnn.DNN_TARGET_MYRIAD)
+
+parser = argparse.ArgumentParser(description='Use this script to run virtial try-on using CP-VTON',
+                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+parser.add_argument('--input_image', '-i', help='Path to image with person. Skip this argument to capture frames from a camera.')
+parser.add_argument('--input_cloth', '-c', help='Path to target cloth image')
+parser.add_argument('--gmm_model', '-gmm', required=True, help='Path to .onnx model.')
+parser.add_argument('--tom_model', '-tom', required=True, help='Path to .onnx model.')
```

Comment:
Please add more information which model is expected (like in case of `segmentation_model`).